tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionTrackManager.cs

```diff
@@ -52,7 +52,8 @@ private ExceptionTrackManager(ExceptionReplaySettings settings)
             _cachedInvalidatedCases = new();
 
             _exceptionProcessorTask = Task.Factory.StartNew(
-                                               async () => await StartExceptionProcessingAsync(_cts.Token).ConfigureAwait(false), TaskCreationOptions.LongRunning)
+                                               () => StartExceptionProcessingAsync(),
+                                               TaskCreationOptions.LongRunning)
```

Comment:
This is a `LongRunning` task, a new thread is created for this one, but in `StartExceptionProcessingAsync` you have a `.ConfigureAwait(false)` so this new thread will be lost at that first Task.WhenAny, because on return the continuation will be schedule to a thread pool thread.